### ISSUE TITLE
Tag library does not register

### ISSUE THREAD

**Issue by axru5812** [GibbsConsulting/django-plotly-dash#12]
Hi!

I tried pip installing django-plotly-dash and have added 
`'django_plotly_dash.apps.DjangoPlotlyDashConfig',` to the settings of the django project,
copied the plotly_apps.py file into my project, imported it in my urls.py and added the `{%load plotly_dash %}` in the template but I'm getting an error when i try to render this template:

```django.template.exceptions.TemplateSyntaxError: 'plotly_dash' is not a registered tag library.```

I'm not sure what I have to do to make sure the tag library is registered?

**Comment by delsim** [comment]
That's an interesting one. What happens if you try to import the tag library from a shell, as in:
`python manage.py shell # run shell instead of runserver`
and then at the command prompt:
`from django_plotly_dash.templatetags import plotly_dash`

Also a quick sanity check (something I always forget) - have you completely restarted the server after adding the django_plotly_dash app?

**Comment by axru5812** [comment]
I did try to restart the server (and even the whole computer) but to no avail. 

I tried running the import in the shell now but i get 
```ModuleNotFoundError: No module named 'django_plotly_dash.templatetags' ```

i can import `django_plotly_dash` without errors though. Running the `dir()` command on the imported module gives these attributes:
```
['DjangoDash',
 '__builtins__',
 '__cached__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__path__',
 '__spec__',
 '__version__',
 'admin',
 'app_name',
 'apps',
 'dash_wrapper',
 'models']
```

**Comment by delsim** [comment]
I just tried installing the package using pip and got the same error.

Looking at the installed package it appears that the templatetags directory is missing, and in turn it looks like the package setup is deficient. I'll prepare a patch and push a release. Thanks for reporting it.

PS one workaround in the interim would be to `git clone` the repo and use it as if developing it, if you're really desperate to get going.

**Comment by GibbsConsulting** [comment]
This issue should be resolved with v0.5.0, which has been released and pushed to PyPI and ReadTheDocs.

There are quite a lot of other changes in this version as we march towards a 1.0 release.
